hibernate-core/src/main/java/org/hibernate/boot/jaxb/internal/XmlSources.java

```diff
@@ -27,7 +27,11 @@
 import org.w3c.dom.Document;
 
 /**
- * Helper for building and handling {@link XmlSource} references
+ * Helper for building and handling {@link XmlSource} references.
```

Comment:
Either add a note to `#addJar` stating that `orm.xml` files are not handled within the jar file, or change `#addJar` to recognize `orm.xml` files also.

Reply:
Unless I am mistaken we never use `#addJar` from JPA bootstrapping.  So this would already be a Hibernate-specific series of calls.

But anyway, it was an either/or statement I made.  I do not think it is a good idea to change the semantic of this - aka just add a note saying `orm.xml` files are not picked up.  

If we want to support `#addJar` + `orm.xml`, probably better at this point to add an overload accepting a boolean.

Reply:
Well, I was wondering if we should add a method something approximately like `addOrmXmlFromJar()` - don't know what to call it, but it would know where to look (in `META-INF`).